src/avio.jl

```diff
@@ -295,7 +295,14 @@ function VideoReader(avin::AVInput, video_stream=1;
                                  width, height, target_format,
                                  transcode_interpolation, C_NULL, C_NULL, C_NULL)
 
-    avpicture_fill(pointer(aTargetVideoFrame), target_buf, target_format, width, height)
+    align = 16
+    offset = 64 # length of data
```

Comment:
Are these values correct across all resolutions/formats etc? Tests seemed to be passing after your first commit, so I guess that's decent evidence..

Reply:
offset = 64 length of [AVFrame.data](https://ffmpeg.org/doxygen/2.7/structAVFrame.html), 8 byte = pointer length in 64 bit system and AV_NUM_DATA_POINTERS=8, thus 64. Maybe I should add 32 bit compatibility. `align=16` is value used by [PyAV](https://github.com/PyAV-Org/PyAV/commit/673f072e6a8552b6c5c8697d9e2d47b65ea9dc34#diff-dc208ab0752ca9c4f2bf1fc7c7fc3efc), they tried custom specifying, forced 8 and finally use forced 16.

Reply:
Thanks for the detail. I think adding in-line comments to explain & a link to the PyAV commit would be helpful, otherwise they're a bit obscure

Reply:
And I just noticed that the 32-bit windows CI is failing. We should make the offset dependent on word size

Reply:
I added `WORD_SIZE` but I don't have 32-bit system to run test locally. Wait for CI result.